core/integration/features/0084-VDPR-013.feature

```diff
@@ -0,0 +1,121 @@
+      | horizon | probability | auction extension |
+      | 3600    | 0.99        | 3                 |
+
+    And the liquidity sla params named "SLA-22":
+      | price range | commitment min time fraction | performance hysteresis epochs | sla competition factor |
+      | 0.5         | 0.6                          | 1                             | 1.0                    |
+
+    And the markets:
+      | id        | quote name | asset | risk model            | margin calculator   | auction duration | fees          | price monitoring | data source config     | linear slippage factor | quadratic slippage factor | sla params |
+      | ETH/MAR24 | ETH        | ETH   | log-normal-risk-model | margin-calculator-1 | 2                | fees-config-1 | price-monitoring | default-eth-for-future | 1e0                    | 0                         | SLA-22     |
+
+    And the following network parameters are set:
+      | name                                                  | value |
+      | market.liquidity.bondPenaltyParameter                 | 0.2   |
+      | validators.epoch.length                               | 5s    |
+      | market.liquidity.stakeToCcyVolume                     | 1     |
+      | market.liquidity.successorLaunchWindowLength          | 1h    |
+      | market.liquidity.sla.nonPerformanceBondPenaltySlope   | 0.7   |
+      | market.liquidity.sla.nonPerformanceBondPenaltyMax     | 0.6   |
+      | validators.epoch.length                               | 10s   |
+      | market.liquidity.earlyExitPenalty                     | 0.25  |
+
+    Given the average block duration is "1"
+  @Now
+  Scenario: 001: Check that the volume discount factor is updated after each epoch
+    Given the parties deposit on asset's general account the following amount:
+      | party  | asset | amount |
+      | lp1    | ETH   | 10000000 |
+      | party1 | ETH   | 10000000 |
+      | party2 | ETH   | 10000000 |
+      | party3 | ETH   | 10000000 |
+
+    And the parties submit the following liquidity provision:
+      | id   | party | market id | commitment amount | fee   | lp type    |
+      | lp_1 | lp1   | ETH/MAR24 | 100000            | 0.02  | submission |
+
+    Then the parties place the following orders:
+      | party  | market id | side | volume | price | resulting trades | type       | tif     |
+      | party1 | ETH/MAR24 | buy  | 10     | 900   | 0                | TYPE_LIMIT | TIF_GTC |
+      | party1 | ETH/MAR24 | buy  | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+      | lp1    | ETH/MAR24 | buy  | 100    | 990   | 0                | TYPE_LIMIT | TIF_GTC |
+      | lp1    | ETH/MAR24 | sell | 100    | 1010  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party2 | ETH/MAR24 | sell | 10     | 1100  | 0                | TYPE_LIMIT | TIF_GTC |
+      | party2 | ETH/MAR24 | sell | 1      | 1000  | 0                | TYPE_LIMIT | TIF_GTC |
+
+    Then the opening auction period ends for market "ETH/MAR24"
```

Comment:
This is fine here, but should be noted that in scenarios with more than 1 market, this will cause problems. It's probably best to use `When the network moves ahead "1" blocks` so it's clear that this step progresses time and therefore affects all markets.